openapi/resource-server.yaml

```diff
@@ -1012,6 +1020,100 @@ components:
         - sentAmount
         - createdAt
         - updatedAt
+    outgoing-payment-with-spent-amounts:
+      title: Outgoing Payment With Grant Spent Amounts
+      description: 'An **outgoing payment** resource represents a payment that will be, is currently being, or has previously been, sent from the wallet address.'
+      type: object
+      examples:
+        - id: 'https://ilp.rafiki.money/outgoing-payments/8c68d3cc-0a0f-4216-98b4-4fa44a6c88cf'
+          walletAddress: 'https://ilp.rafiki.money/alice/'
+          failed: false
+          receiver: 'https://ilp.rafiki.money/aplusvideo/incoming-payments/45d495ad-b763-4882-88d7-aa14d261686e'
+          receiveAmount:
+            value: '2500'
+            assetCode: USD
+            assetScale: 2
+          debitAmount:
+            value: '2600'
+            assetCode: USD
+            assetScale: 2
+          sentAmount:
+            value: '2500'
+            assetCode: USD
+            assetScale: 2
+          grantSpentDebitAmount:
+            value: '2600'
+            assetCode: USD
+            assetScale: 2
+          grantSpentReceiveAmount:
+            value: '2500'
+            assetCode: USD
+            assetScale: 2
```

Comment:
I think these properties should be optional, since in the future, we will only allow either passing in `limits.receiveAmount` or `limits.debitAmount`, but not both: 
- https://github.com/interledger/open-payments/issues/471

Likewise, this would mean that in the example, only one or the other property will be present.

Reply:
made them optional and returned the 2nd examples to show one or the other